flepimop/gempyor_pkg/src/gempyor/flepimop_push.py

```diff
@@ -0,0 +1,164 @@
+import os
+import click
+import shutil
+from gempyor.file_paths import create_file_name_for_push
+
+
+@click.command()
```

Comment:
Just out of curiosity why use the `click` package over python's `argparse`? My high level view has been that `click` just adds syntactic sugar for things that `argparse` already does at the expense of adding a new dependency to keep up with and a new set of functions/APIs to learn.

Reply:
update to this note - I think both @TimothyWillard and I are on board with the click approach.

That said, I think we might want to re-think the approach here a little. I'll elaborate in a comment against broad PR.

Reply:
The reason for click package  is that we also use `click ` in file calibrate.py, cli.py.  I think this may be our preferred way to read arguments. If not, I can switch to `argparse`.

Reply:
We are agreed with click.